### DISCUSSION TITLE
Move cursor by word on Mac - cant get Option+arrow working

### DISCUSSION THREAD

**Discussion by dabeeeenster** [wezterm/wezterm#1094] [Q&A] [answered]
Hi,

Am on a Mac and trying to get Wezterm to behave like my existing iTerm2 and Terminal. Both of those allow me to move by word with the left-option + left/right cursor key. If I hit this key combo in Wezterm, I get `D` printed on the screen. I cant figure out from the docs how to set the right bind? In iTerm2 I have "Left Option sends Esc+" 

Any ideas how to do this? I don't have any key bind related stuff in my `wezterm.lua`.

**Comment by wez** [comment, marked as answer]
Looking at the Terminal.app preferences:

![image](https://user-images.githubusercontent.com/117777/131708640-c5650a36-f266-4f33-ba9e-550f185c746f.png)

Option left and right are rebound to `\033b` and `\033f` respectively.

You use the `SendString` action to rebind keys, and if you look at the docs for SendString, you'll see an example that covers exactly these key bindings:

https://wezfurlong.org/wezterm/config/lua/keyassignment/SendString.html

**Comment by dabeeeenster** [comment]
Perfect thanks so much for the reply
